data/common.yaml

```diff
@@ -62,6 +62,7 @@ rabbitmq::ssl_fail_if_no_peer_cert: false
 rabbitmq::ssl_management_verify: 'verify_none'
 rabbitmq::ssl_management_fail_if_no_peer_cert: false
 rabbitmq::ssl_versions: ~
+rabbitmq::ssl_client_renegotiation: ~
```

Comment:
We don't need this default value, because we've the same value in init.pp (which is also the preferred place, because puppt-strings will pick it up and document it in the REFERENCE.md)

Reply:
updated in second commit.